ansible/roles_ocp_workloads/ocp4_workload_rhte19_shadowman/tasks/remove_workload.yml

```diff
@@ -0,0 +1,15 @@
+---
+# Implement your workload removal tasks here
+# ------------------------------------------
+
```

Comment:
Is this a per-user workload? If so removal tasks should be implemented to clean up when the user is removed.